etc/openstack_deploy/group_vars/all/osa.yml

```diff
@@ -71,6 +71,50 @@ glance_glance_api_conf_overrides:
   DEFAULT:
     image_cache_dir: None
 
+# Octavia tuning
+# activate octavia - Queens only
+octavia_v2: True
+octavia_v1: False
+# Keep this while there is no Barbican or Vault
+octavia_tls_listener_enabled: False
+
+# Octavia HA settings
+octavia_loadbalancer_topology: ACTIVE_STANDBY
+octavia_spare_amphora_pool_size: 0
+octavia_enable_anti_affinity: "{{ lookup('env', 'DEPLOY_AIO') != 'yes' }}"
+
+# no event streamer
+octavia_event_streamer: False
+octavia_sync_provisioning_status: False
+
+rackspace_octavia_files_folder: "/opt/rpc-openstack/playbooks/templates/octavia"
```

Comment:
why is this needed? (and where is it used)?

Reply:
Two lines below this one.